tests/serverpod_test_server/test_integration/database_operations/unsafe_query/database_parameter_binding_test.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  var session = await IntegrationTestServer().session();
+
+  group(
+      'Given a simple data object in database when fetching with query using positional parameter binding',
+      () {
+    late SimpleData entry_1;
+    late SimpleData entry_2;
+    late DatabaseResult all_rows;
+    late DatabaseResult list_param;
+    setUpAll(() async {
+      entry_1 = await SimpleData.db.insertRow(session, SimpleData(num: 1));
+      entry_2 = await SimpleData.db.insertRow(session, SimpleData(num: 2));
+      all_rows = await session.db.query(
+        r'SELECT * FROM simple_data WHERE id IN ($1, $2);',
+        parameters: QueryParameters.positional([entry_1.id, entry_2.id]),
+      );
+      list_param = await session.db.query(
+        r'SELECT * FROM simple_data WHERE id=$1;',
+        parameters: QueryParameters.positional([entry_1.id]),
+      );
+    });
+
+    tearDownAll(() async {
+      await SimpleData.db
+          .deleteWhere(session, where: (t) => Constant.bool(true));
+    });
+
+    test('then all_rows reports the correct number of rows.', () async {
+      expect(all_rows.length, 2);
+    });
```

Comment:
Suggestion: Break this scenario out with a different `When` description section.

Almost there, but we are still not getting the full context of the test from the description.

A good composed test description could be "Given a simple data object in database when fetching with query using **multiple** positional parameter bindings then selected rows are returned." as opposed to the other scenario that could have a description "Given a simple data object in database when fetching with query using **single** positional parameter bindings then selected row is returned.".

Reply:
Make this change to the rest of the tests as well.